docs_user/modules/proc_adopting-telemetry-services.adoc

```diff
@@ -99,8 +115,12 @@ sources:
         - volume.size
```

Comment:
This is not the correct pollsters file that the user would encounter in ceilometer-central-agent. The file default enabled pollsters will be:
        - volume.*
        - image.*

And if the user has already deployed swift in its environment, this extra one will be added:
         - storage.*